### DISCUSSION TITLE
Can no longer compile using "Release" in visual studio 2022

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4423] [Q&A] [answered]
It compiles using debug mode. What might I need to look for/change?
Missing .starts_with 
![image](https://github.com/xLightsSequencer/xLights/assets/4643499/9311d17c-9a0c-49ce-a696-df7a275067ee)

**Comment by AzGilrock** [comment]
I pulled and built tonight.  Have you updated to the latest wxWidgets called out in the Readme?

**Reply by merryoncherry** [reply]
C++20.  Dan put starts_with and it broke the build, rather than changing the code apparently we switch to C++20 real quick.

**Comment by derwin12** [comment]
Ok.. I will Google that up...

**Comment by AzGilrock** [comment]
So maybe its a project setting if your in Visual Studio?

**Comment by derwin12** [comment, marked as answer]
Yep.. got it.. Thanks folks for pointing me in the right direction.

![image](https://github.com/xLightsSequencer/xLights/assets/4643499/e5514b03-c788-4eec-be37-04db43567b70)

**Comment by derwin12** [comment]
Similar issue with my github's nightly build job. I will have to track that down..
